src/App.vue

```diff
@@ -13,19 +13,36 @@
           </label>
           <label class="flex items-center mt-3 border px-3 py-2 cursor-pointer rounded-md font-semibold" :class="{ 'btn-selected': formData.isImportFromCsv === true }">
             <input type="radio" name="animation" class="hidden form-radio border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50" @click="() => formData.isImportFromCsv = true">
-            <span>Import from CSV</span>
+            <sformData.promptconst pan>Import from CSV</span>
```

Comment:
Not that this matters since it is in a comment, but was this an accidental copy/paste? If so, let's just clean it back up to how it was

Reply:
Opps your right didn't even realize that.